io.github.mfat.sshpilot.yaml

```diff
@@ -0,0 +1,173 @@
+app-id: io.github.mfat.sshpilot
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+command: sshpilot
+finish-args:
+  - --share=network
+  # - --share=ipc
+  - --filesystem=~/.ssh
+  - --filesystem=~/.config/sshpilot
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=ssh-auth
+  #- --device=dri
+  - --persist=.ssh
+  - --persist=.config
+  #- --filesystem=/tmp
+  - --filesystem=xdg-run/gvfs:rw
+  - --filesystem=xdg-run/gvfsd
+  - --talk-name=org.freedesktop.secrets
```

Comment:
Why is this needed? Also consider using the portal. If you use something libsecret based it uses the portal already